samples/Humidity_DHT22/app/application.cpp

```diff
@@ -47,9 +47,9 @@ void init()
 	if(dht.readTempAndHumidity(th))
 	{
 		Serial.print("\tHumidity: ");
-		Serial.print(h);
+		Serial.print(th.humid);
```

Comment:
Please, split the PR into smaller PRs. I can see that there are changes related to:
* APA 102 LED
* ArduCAM
* DHT22
etc..